etc/chat-core.api.md

```diff
@@ -17,12 +17,10 @@ export interface ChatConfig {
     version?: "STAGING" | "PRODUCTION" | number;
 }
 
-// @public
-export class ChatCore {
-    constructor(chatConfig: ChatConfig, promptPackage?: ChatPrompt);
-    getNextMessage(request: MessageRequest): Promise<MessageResponse>;
-    streamNextMessage(request: MessageRequest): Promise<StreamResponse>;
-}
+// Warning: (ae-forgotten-export) The symbol "ChatCore_2" needs to be exported by the entry point index.d.ts

```

Comment:
seems like the class needs to be exported anyway

Reply:
rip, updating